apps/api/src/secret/service/secret.service.ts

```diff
@@ -490,6 +490,35 @@ export class SecretService {
 
     return response
   }
+  async getRevisionsOfSecret(
+    user: User,
+    secretId: Secret['id'],
```

Comment:
Could you please add pagination to this? Rest looks good :)

Reply:
Yeah sure.I could add pagination offered by nestjs.But it will require page limit and page numbers as query params ig,will it be okay then ?

Reply:
@rajdip-b can you weigh in your opinion on this

Reply:
Hey! I was AFK. Okay so, you can refer to the other controllers that use pagination - you can check how we do the query parameters in controllers and how we do the implementation in the service layer. That would get you started :D

Reply:
Yeah,so I'd definitely have to use query params ig,so the req GET /api/secret/:secretId/revisions/:environmentId ,would also have an addtional query param then.Thats okay right?

Reply:
Service layer - https://github.com/keyshade-xyz/keyshade/blob/b272df3210e7338be0722d6711cf110998500cb7/apps/api/src/secret/service/secret.service.ts#L494
Controller layer - https://github.com/keyshade-xyz/keyshade/blob/b272df3210e7338be0722d6711cf110998500cb7/apps/api/src/secret/controller/secret.controller.ts#L69

Reply:
Hope this helps!

Reply:
umm will i be needing the params sort and order ? the revisions are already in desc order right

Reply:
Uh, no. You would need to explicitly set that.

Reply:
hey I made the changes check it out.Btw there were some merge conflicts too.Do check if theres anything wrong